src/components/data/TableRow.js

```diff
@@ -0,0 +1,63 @@
+import classnames from 'classnames';
+import { useContext } from 'preact/hooks';
+
+import { useSyncedRef } from '../../hooks/use-synced-ref';
+import { downcastRef } from '../../util/typing';
+
+import TableSectionContext from './TableSectionContext';
+import TableContext from './TableContext';
+
+/**
+ * @typedef {import('../../types').PresentationalProps} CommonProps
+ * @typedef {Omit<import('preact').JSX.HTMLAttributes<HTMLElement>, 'size'>} HTMLAttributes
+ *
+ * @typedef TableRowProps
+ * @prop {boolean} [selected=false]
+ */
+
+/**
+ * Render a table row
+ *
+ * @param {CommonProps & TableRowProps & HTMLAttributes} props
+ */
+const TableRowNext = function TableRow({
+  children,
+  classes,
+  elementRef,
+
+  selected,
+
+  ...htmlAttributes
+}) {
```

Comment:
Do you know off-hand why this would be the case? We're using `aria-selected` and `role="grid"` on the `table` element. I can do some digging but let me know if you have info.

Reply:
I don't know why this would happen, so please dig away👷‍♀️.